### PR TITLE
Oncoprint: do not unnecessarily re-render clinical dropdown, losing scroll position

### PR DESCRIPTION
**Pull request by adamabeshouse**
fixes #7426

### CODE REVIEW

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 1]
```diff
@@ -760,6 +760,28 @@ export class ResultsViewPageStore {
         },
     });
 
+    @computed.struct get comparisonGroupsReferencedInURL() {
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts:763]: hey, can you explain struct to me?  i looked it up.   it says "ignore new values that are structurally equal" but i don't quite understand what structurally equal means?

**Comment by alisman** [comment]
Curious why the study view files are involved

**Comment by adamabeshouse** [comment]
@alisman the study view components are used in the AddClinicalTracks component - they're just the tabs of the dropdown.

File: src/pages/resultsView/ResultsViewPageStore.ts (hunk 1 above)
Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStore.ts:763]: somethings lost in translation there but it means deep equal

Comment (alisman) [reply src/pages/resultsView/ResultsViewPageStore.ts:763]: got it.  thanks.   i guess struct comes from structurally equal

**Pull request merged**